packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -1471,7 +1471,12 @@ namespace Ifpack2 {
       using impl_type = ImplType<MatrixType>;
       using local_ordinal_type_1d_view = typename impl_type::local_ordinal_type_1d_view;
       using size_type_1d_view = typename impl_type::size_type_1d_view;
-      using impl_scalar_type_1d_view_tpetra = typename impl_type::impl_scalar_type_1d_view_tpetra;
+      using scalar_1d_managed_view_tpetra = typename impl_type::impl_scalar_type_1d_view_tpetra;
+      using impl_scalar_type_1d_view_tpetra = Kokkos::View<
+        typename scalar_1d_managed_view_tpetra::data_type,
+        typename scalar_1d_managed_view_tpetra::array_layout,
+        typename scalar_1d_managed_view_tpetra::device_type,
+        Kokkos::MemoryTraits<Kokkos::Unmanaged>>;
```

Comment:
I recommend to use 
```
using impl_scalar_type_1d_view_tpetra = typename impl_type::impl_scalar_type_1d_view_tpetra;
using impl_scalar_type_1d_view_tpetr_unmanaged = Unmanaged<impl_scalar_Type_1d_view_tpetra>;
```

Reply:
That's great, I didn't see that at the top.